FAQ-LICENSING.md

```diff
@@ -0,0 +1,47 @@
+# JTS License FAQ
+
+JTS is dual-licensed under the EDL and EPL 
+as described [here](LICENSES.md).
+
+This FAQ clarifies the implications of the JTS software licensing.
+
+## Terminology
+
+* **Downstream projects** are codebases which link to JTS libraries
+  * e.g. GeoTools, GeoServer, etc.
+* **Derivative projects** are ports of JTS to other languages 
+  * e.g. GEOS, JSTS, etc.
+
+
+## Licensing Frequently Asked Questions
+
+**A1. What is the difference between EDL and BSD-3?**
+
+  BSD-3 is a license *family*.  EDL is the actual license and is called that because Eclipse is the distributing organization.
+
+**A2. JTS no longer uses LGPL.  What does this mean for downstream projects?**
+
+JTS is now released under a dual license: EDL and EPL.  
+Dual-licensing gives downstream projects more options:
+
+* The **EDL** is a BSD-3-style license, which is more permissive than the LGPL.  
+  This allows a wider variety of uses; for instance, it permits iOS use of JTS for mobile development.
+  
+* The **EPL** provides a similar permissiveness to LGPL with different wording around patents
+
+**A3. But I thought LGPL and EPL do not mix? How does this effect derivative projects (like the LGPL GEOS project)?**
+
+An LGPL project (like GEOS) should use a BSD-3-style License (the BSD-3 License is compatible with the LGPL license used by GEOS).
```

Comment:
The "BSD-3-style License" that JTS is released under is the EDL, right? Would it be correct to say:

"An LGPL project (like GEOS) should use JTS under the terms of EDL, which is compatible with the LGPL." ?

If correct, this could also avoid one misinterpreting the FAQ as suggesting that GEOS should switch its own license to BSD.

Reply:
That does sound more like the intention to me - I was wondering about the original wording myself.  @jodygarnett can you confirm this reflects the intention of this point?